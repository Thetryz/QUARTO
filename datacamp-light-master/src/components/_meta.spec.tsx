import * as React from "react";
import { shallow } from "enzyme";

describe("Enzyme", () => {
  it("should render an empty div", () => {
    const wrapper = shallow(<div />);
    expect(wrapper.find("div").length).toBe(1);
  });

  it("should create and match snapshot", () => {
    const wrapper = shallow(<div />);
    expect(wrapper).toMatchSnapshot();
  });
});
