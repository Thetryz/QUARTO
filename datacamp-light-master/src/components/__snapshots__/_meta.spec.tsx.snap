// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Enzyme should create and match snapshot 1`] = `<div />`;
